src/components/ProjectsPage.js

```diff
@@ -1,83 +1,121 @@
 import React from "react";
+import { Button, Row, Col, Card, Container } from 'react-bootstrap';
```

Comment:
do you mind changing this whole file to use 2 space tabbing instead of 4?

Reply:
done.